src/main.ts

```diff
@@ -296,6 +296,7 @@ export default class MarkdownAttributes extends Plugin {
                 return tr.effects.reduce((deco, effect) => {
                     if (effect.is(replace))
                         return effect.value.update({
+							// @ts-ignore
```

Comment:
Why do we need to ignore here

Reply:
to shut up vs code while i patched things up